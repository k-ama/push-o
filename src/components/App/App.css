.App {
  display: flex;
  position: relative;
  flex-direction: column;
  height: 100%;
}

