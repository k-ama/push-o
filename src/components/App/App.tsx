import React from 'react';
import './App.css';
import Counter from '../Counter/Counter';
import Toggle from '../Toggle/Toggle'

const blue = {
  r:70,
  g:130,
  b:180
}

const pink = {
  r:141,
  g:2,
  b:31
}

function App() {
  return (
    <div className="App">
      <div>
        <Counter
          minColor={blue}
          maxColor={pink}
        />
      </div>
      <div>
         <Toggle/>
      </div>
    </div>
  );
}

export default App;
