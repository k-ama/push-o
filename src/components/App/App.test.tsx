import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

jest.mock("../Counter/Counter")
jest.mock("../Toggle/Toggle")

describe('App', () => {

  describe('Snapshot', () => {
    test('renders App with no props', () => {
      const component = renderer.create(<App />)
      expect(component.toJSON()).toMatchSnapshot()
    })
  })

})
