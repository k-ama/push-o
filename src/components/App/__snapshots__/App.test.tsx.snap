// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot renders App with no props 1`] = `
<div
  className="App"
>
  <div>
    <div
      className="Counter"
    />
  </div>
  <div>
    <div
      className="Toggle"
    />
  </div>
</div>
`;
